src/utils/useVideoTiming.ts

```diff
@@ -0,0 +1,99 @@
+import { useState, useEffect } from "react";
+import { TARGET_TYPE } from "../constants";
+import { getVideoCurrentTimeInMilliseconds } from "./util";
+
+export interface UseVideoTimingProps {
+    targetType: typeof TARGET_TYPE.PAGE | typeof TARGET_TYPE.FRAME;
+    referenceEl?: HTMLElement;
+    activeAnnotationId: string | null;
+    annotations: Array<{ id: string; target: { location: { value: number } } }>;
+}
+
+export interface UseVideoTimingReturnType {
+    isVideoSeeking: boolean;
+    targetVideoTime: number | null;
```

Comment:
can we just use `targetVideoTime?` here? It would also simplify the useState by not having to feed it a default value

Reply:
Thanks for noticing this. I actually don't even need to return that. That value is not used outside of the hook. Removed it and updated the tests.